docs/network-policy-recommendation.md

```diff
@@ -0,0 +1,132 @@
+# Network Policy Recommendation
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Deployment](#deployment)
+- [Perform NetworkPolicy Recommendation](#perform-networkpolicy-recommendation)
+  - [Run a policy recommendation job](#run-a-policy-recommendation-job)
+  - [Check the status of a policy recommendation job](#check-the-status-of-a-policy-recommendation-job)
+  - [Retrieve the result of a policy recommendation job](#retrieve-the-result-of-a-policy-recommendation-job)
+<!-- /toc -->
+
+## Introduction
+
+Theia NetworkPolicy Recommendation recommends the NetworkPolicy configuration to secure Kubernetes
+network and applications. It analyzes the network flows collected by
+[Grafana Flow Collector](network-flow-visibility.md#grafana-flow-collector) to generate
+[Kubernetes NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+or [Antrea NetworkPolicies](https://github.com/antrea-io/antrea/blob/main/docs/antrea-network-policy.md).
+The goal of this feature is to aid the cluster admin or developers to secure
+their applications based on the idea of the Zero Trust model.
+
+## Deployment
+
+To use the NetworkPolicy recommendation feature, please follow this [doc](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md) to deploy Flow Exporter and Flow Aggregator first.
```

Comment:
Copy my comments here too:

> We received a valid feedback the deployment steps are too complex to follow, and can be a big challenge for a normal user. Could we have an all-in-one quick-start to deploy everything, rather than refer to multiple docs? At least, we should have a quick-start to deploy Grafana Flow Collector with default settings, and then this doc can point to that.

Reply:
For your reference, this is the quick-start we are adding for multi-cluster (yes it still looks complex without any automation): https://github.com/antrea-io/antrea/pull/3853#diff-af241a265571db5ff3e18981d0ddfe1cef2dd672c3a1815c89164245f0906661

Reply:
Thank Jianjun, for the quick-start, I think using helm is the simplest way to achieve that. After https://github.com/antrea-io/theia/pull/31 is merged, all the needed steps will be:
1. Deploy Antrea with Flow Exporter, and Flow Aggregator following a doc inside Antrea repo.
2. Run the helm install command of Theia to deploy Clickhouse, grafana, spark operator, etc.
Is this good enough now? Or we'd better to come up a solution to deploy all components in one step?

cc @salv-orlando

Reply:
@dreamtalen - as a user, ideally I would be able to deploy antrea + theia by:
1) setting mandatory parameters in values.yaml
2) do something like `helm install theia && helm install antrea` (or the other way around)

So, your plan looks quite good, but in the quick start doc I think it's better to provide users a way to do things very quickly only using helm (e.g.: not 1) set config values for antrea; 2) deploy antrea with helm; 3) do some config changes; 4)set config values for Theia;  5) deploy theia with helm